src/script/tracking/Countly.helpers.ts

```diff
@@ -17,9 +17,17 @@
  *
  */
 
+import {getWebEnvironment} from 'Util/Environment';
+
 import {Config} from '../Config';
 
 export function isCountlyEnabledAtCurrentEnvironment(): boolean {
+  const {isDev, isEdge, isInternal, isLocalhost, isStaging} = getWebEnvironment();
+
+  if (isDev || isEdge || isInternal || isLocalhost || isStaging) {
+    return true;
+  }
```

Comment:
Can be simplified like that? :)

```suggestion
  if (!isProduction) {
    return true;
  }
```

Reply:
Not the same